scripts/test-runner.js

```diff
@@ -44,34 +44,71 @@ const isAddressReachable = (host, port, timeoutMs) => {
 const startRC = async (background) => {
     if (ON_WINDOWS) {
         if (background) {
-            rcProcess = spawn('start /min "hazelcast-remote-controller" cmd /c '
-                + `java -Dhazelcast.enterprise.license.key=${HAZELCAST_ENTERPRISE_KEY} -cp ${CLASSPATH} `
-                + 'com.hazelcast.remotecontroller.Main --use-simple-server > rc_stdout.txt 2> rc_stderr.txt', [], {
-                stdio: ['ignore', 'inherit', 'inherit'],
+            rcProcess = spawn(
+                'start /min "hazelcast-remote-controller" cmd /c '
+              + `java -Dhazelcast.enterprise.license.key=${HAZELCAST_ENTERPRISE_KEY} -cp ${CLASSPATH} `
+              + 'com.hazelcast.remotecontroller.Main --use-simple-server > rc_stdout.txt 2> rc_stderr.txt', [], {
+                stdio: 'ignore',
                 shell: true
             });
         } else {
-            rcProcess = spawn(`java -Dhazelcast.enterprise.license.key=${HAZELCAST_ENTERPRISE_KEY} -cp ${CLASSPATH} `
-                + 'com.hazelcast.remotecontroller.Main --use-simple-server > rc_stdout.txt 2> rc_stderr.txt', [], {
-                stdio: ['ignore', 'inherit', 'inherit'],
-                shell: true
-            });
+            try {
```

Comment:
The mocha process was not killing rc process on exit even if I add a handler. After changing the spawn type from shell to non shell mode, it works